### PR TITLE
Add community members to mentionable list

### LINKED ISSUE #12911: Add community member to mentionable contacts
# Problem
When accepting a new member in a community, that member can currently only be mentioned if they are a contact or if they have send a message in the chat. 

As a community owner I want to mention newly added community members (that are not a contact and haven't send a message to a chat in a community), because I want to welcome and introduce them upon accepting them as a member.

## Implementation
Add community members to mentionable users

## Acceptance Criteria
I can mention a user that is not a contact after accepting their request to join a community

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-react/issues/12911

### Summary
PR implements a feature, which allows users to mention newly joined community members. Community members will be shown in the community chat mention suggestion list, even if they are not their contact.

status: ready

### CODE REVIEW

File: src/status_im/notifications_center/core.cljs [hunk 1]
```diff
@@ -19,15 +19,16 @@
...
-    {:db (-> db
-             (assoc-in [:activity.center/notifications :notifications] notifications)
-             (assoc :activity.center/notifications-count (max 0 unread-count)))
-     :dispatch (cond
-                 (= (:view-id db) :notifications-center)
-                 [:mark-all-activity-center-notifications-as-read]
-
-                 (= (:view-id db) :chat)
-                 [:accept-all-activity-center-notifications-from-chat (:current-chat-id db)])}))
+    (merge
```
Comment (Parveshdhull) [inline src/status_im/notifications_center/core.cljs:22]: this part is not related to this issue. it will remove null dispatch error message from the metro. 
is it ok to keep this in this PR or create a separate one?

**Review by bitgamma — APPROVED** [review]

File: src/status_im/chat/models/mentions.cljs [hunk 2]
```diff
@@ -173,8 +173,10 @@
 (defn get-mentionable-users
   [{{:keys          [current-chat-id]
      :contacts/keys [contacts] :as db} :db}]
-  (let [{:keys [chat-type users] :as chat}
+  (let [{:keys [chat-type community-id users] :as chat}
         (get-in db [:chats current-chat-id])
+        mentionable-commmunity-members @(re-frame/subscribe
```
Comment (rasom) [inline src/status_im/chat/models/mentions.cljs]: Is that an appropriate way to use subs? I know I did this once with mentions as well, but as I remember @flexsurfer was strongly against it.

Comment (Parveshdhull) [reply src/status_im/chat/models/mentions.cljs]: Hi @rasom,
Yes you are right, I can create a function that will give mentionable members list.
And I can use that function here and also in subs.
Is that you meant?
Thanks

Comment (rasom) [reply src/status_im/chat/models/mentions.cljs]: @Parveshdhull , yes, something like this

**Comment by churik** [comment]
e2e failures are not related to PR.
Tested mentionable list for group chats, public, 1-1, communities, LGTM!

**Devices:**
- Samsung Galaxy S21 (Android 11)
- Xiaomi Mi Note 9 Pro (Android 10)
